src/Components/WorkExpereince.js

```diff
@@ -0,0 +1,293 @@
+          </p>
+        </Tab>
+        <Tab eventKey="st-electronic" title="ST Electronics">
+          <h2 style={{ color: "#0047AB", fontWeight: "bold" }}>
+            Software Development Intern{" "}
+            <span style={{ color: "red", fontWeight: "bold" }}>@</span>{" "}
+            <span style={{ color: "magenta", fontWeight: "bold" }}>
+              ST Electronics
+            </span>
+          </h2>
+          <span style={{ fontSize: "max(1.2vw, 16px)", color: "aquamarine" }}>
+            From Mar 2017 to May 2017
+          </span>
+          <p>
+            <ul>
+              <li>
+                Provided cybersecurity professionals with a conducive
+                environment to enhance their skills through the practical
+                application of knowledge.
+              </li>
+              <li>
+                Assisted in STECA's deep domain expertise and operational
+                experience
+              </li>
+              <li>
+                Imparted realistic cybersecurity training to the trainees using
+                real-life use cases to correlate with key concepts of
+                cybersecurity.
+              </li>
+            </ul>
+          </p>
+        </Tab>
+        <Tab eventKey="steca" title="STECA">
+          <h2 style={{ color: "#0047AB", fontWeight: "bold" }}>
+            Certified Cyber Security Operations Specialist{" "}
+            <span style={{ color: "red", fontWeight: "bold" }}>@</span>{" "}
+            <span style={{ color: "magenta", fontWeight: "bold" }}>STECA</span>
+          </h2>
+          <span style={{ fontSize: "max(1.2vw, 16px)", color: "aquamarine" }}>
+            From Mar 2017 to May 2017
+          </span>
+          <p>
+            <ul>
+              <li>
+                Applied best practices in cyber defence skills to detect,
+                contain, eradicate and report on successful cyber breaches.
+              </li>
+              <li>
+                Understood the team-based, communication and reporting skills
+                required to deliver an effective and efficient response to a
+                cyber breach.
+              </li>
+              <li>
+                Evaluated on explaining the entire kill-chain of various
+                cyber-attacks.
+              </li>
+            </ul>
+          </p>
+        </Tab>
+      </Tabs>
+    </Container>
+  );
+}
+
+function PC() {
+  return (
+    <Tab.Container defaultActiveKey="first">
+      <Container
+        id="work"
+        style={{
+          color: "#20C20E",
+          fontFamily: "Share Tech Mono",
+        }}
+      >
+        <h1
+          style={{
+            fontSize: "max(3vw, 32pt)",
+            fontFamily: "Rubik Mono One",
+          }}
+        >
+          4.Professional Work Experience
+        </h1>
+        <Row style={{}}>
+          <Col sm={3} style={{}}>
+            <Nav variant="pills" className="flex-column">
+              <Nav.Item>
```

Comment:
use a map here, to make it less bothersome to write up all this repetitive code.